R/cff2r.R

```diff
@@ -0,0 +1,104 @@
+#' @title Convert from CFF into R
+#' @description Converts a CFF file into DESCRIPTION
+#' @param cffFile Path and name of the CFF file
+#' @param export if `TRUE`, the output is saved as DESCRIPTION (plus an extension to avoid overwriting)
+#' @param ... other arguments passed to functions (e.g.: \code{outfile})
```

Comment:
As a user I'm a little confused by this, because I cannot know what arguments these other functions accept without looking at the source code. In this case it's only 2, so I'd just add them here explicitly and document them properly (assuming `outfile` is `exportDESCRIPTION`).

Reply:
I agree, will commit another suggestion.